### PR TITLE
Show MTA-STS result in scan results

### PR DESCRIPTION
**Pull request by vbrown608**
Depends on #262 

The previous approach of templating most things out with Hugo at build time and then show/hiding them with jQuery based on the scan result was starting to work pretty badly, so I added a lightweight templating library. I think this could be used elsewhere if we make substantial changes to other parts of the results at some point!

### CODE REVIEW

**Comment by lschatzkin** [comment]
I see your note that this depends on #262 , which is closed.  I'm getting an error that says: 
cp: cannot stat 'node_modules/handlebars/dist/handlebars.runtime.min.js': No such file or directory
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

**Comment by vbrown608** [comment]
@lschatzkin could you try running `yarn install`?

**Comment by lschatzkin** [comment]
I'm still trying to get my sandbox working on my loaner, but I still can't get the scanner to work. This is the errors I'm getting: 
`app_1            | 2019/02/21 21:45:26 couldn't connect to mailserver: multiple errors:
postgres_1       | 2019-02-21 21:44:54.124 UTC [24] LOG:  database system was shut down at 2019-02-21 21:44:17 UTC
postgres_test_1  | 2019-02-21 21:44:55.191 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
app_1            | expected environment variable SMTP_USERNAME to be set
postgres_1       | 2019-02-21 21:44:54.135 UTC [1] LOG:  database system is ready to accept connections
postgres_test_1  | 2019-02-21 21:44:55.216 UTC [24] LOG:  database system was shut down at 2019-02-21 21:44:17 UTC
app_1            | expected environment variable SMTP_PASSWORD to be set
app_1            | expected environment variable SMTP_ENDPOINT to be set
app_1            | expected environment variable SMTP_PORT to be set
app_1            | expected environment variable SMTP_FROM_ADDRESS to be set
postgres_test_1  | 2019-02-21 21:44:55.222 UTC [1] LOG:  database system is ready to accept connections
app_1            | expected environment variable FRONTEND_WEBSITE_LINK to be set
app_1            | 2019/02/21 21:45:26 ======NOT SENDING EMAIL======
`

**Comment by lschatzkin** [comment]
Hmm, let's display those errors better:
app_1            | 2019/02/21 21:45:26 couldn't connect to mailserver: multiple errors:
postgres_1       | 2019-02-21 21:44:54.124 UTC [24] LOG:  database system was shut down at 2019-02-21 21:44:17 UTC
postgres_test_1  | 2019-02-21 21:44:55.191 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
app_1            | expected environment variable SMTP_USERNAME to be set
postgres_1       | 2019-02-21 21:44:54.135 UTC [1] LOG:  database system is ready to accept connections
postgres_test_1  | 2019-02-21 21:44:55.216 UTC [24] LOG:  database system was shut down at 2019-02-21 21:44:17 UTC
app_1            | expected environment variable SMTP_PASSWORD to be set
app_1            | expected environment variable SMTP_ENDPOINT to be set
app_1            | expected environment variable SMTP_PORT to be set
app_1            | expected environment variable SMTP_FROM_ADDRESS to be set
postgres_test_1  | 2019-02-21 21:44:55.222 UTC [1] LOG:  database system is ready to accept connections
app_1            | expected environment variable FRONTEND_WEBSITE_LINK to be set
app_1            | 2019/02/21 21:45:26 ======NOT SENDING EMAIL======

**Comment by lschatzkin** [comment]
Do we have a URL to test that will show the MTA results? I'm not seeing them yet.

**Comment by lschatzkin** [comment]
Remove <li> dot when there is only one item in list.

File: js/templates/mta-sts-result.handlebars [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      <p>
+        MTA-STS is a standard that will make it more difficult to perform downgrade attacks on communications with your mailserver.
+      </p>
+
+      {{#if policy}}
+        <p>We found the following{{#ifCond status '===' 0}} valid{{/ifCond}} policy on your mailserver:</p>
+        <pre id="policy-text">{{policy}}</pre>
+
+        {{#ifCond all_messages.length '>' 0}}
+          <h5>Do you manage this email server?</h5>
```
Comment (sydneyli) [inline js/templates/mta-sts-result.handlebars]: Let's add something like, "Here's [a/some] suggestion(s) we have for your deployment:" for warnings and "We found these/this problem(s) with your MTA-STS deployment:" for failures!

Comment (vbrown608) [reply js/templates/mta-sts-result.handlebars]: Great! I wound up adding the framing text to the single message and multiple messages versions, it seemed to work better with the example you sent :)

**Pull request merged**